tests/features/joinup_core/owner_deletion.feature

```diff
@@ -0,0 +1,63 @@
+@api
+Feature: As a site owner
+  In order to always have an owner assigned to a group
```

Comment:
I think it would maybe be clearer if you say

> In order to avoid orphaned groups

because this doesn't explain why it is important that a group needs an owner